packages/client/modules/email/components/TeamInvite.tsx

```diff
@@ -119,23 +126,31 @@ const TeamInvite = (props: TeamInviteProps) => {
             'Parabol is software for remote teams to run online retrospective and check-in meetings. See the video and links below:'
           }
         </p>
-        <a href={ExternalLinks.GETTING_STARTED_RETROS} style={videoLinkStyle} title='Retro 101'>
+        <a
+          href={makeAppURL('https://www.parabol.co/', 'resources/retrospective-meetings', options)}
```

Comment:
-1 I would prefer if we continue using the constants, thus if we change the marketing page we don't forget to update it here. You can just put a helper inside this file
```ts
function appendUTM(url: string) {
  const newUrl = new URL(url)
  Object.entries(urlParams).forEach(([name, value]) => {
    newUrl.searchParams.append(name, value)
  })
  return newUrl.toString()
}
```
and call it with 
```ts
            href={appendUTM(ExternalLinks.GETTING_STARTED_RETROS)}
```